Applications/PlazaDetails/PublishService.qmd

```diff
@@ -9,88 +9,82 @@ Users can publish their algorithm as a service in the openEO Algorithm Plaza. Th
 
 | Field | Required | Description |
 | --- | --- | --- |
-| Service name | Yes | Your service's title that will be displayed in the openEO algorithm plaza. |
-| Summary | No | Short description of your service. The summary will be visible in the openEO algorithm plaza overview.|
-| Description | No | In your service description, begin with an algorithm overview, followed by a brief methodology. It is helpful to list required parameters and include examples and output images for user guidance.|
-| Avatar | No | URL to an image that can be used an avatar of your service. The avatar will be visible in the openEO algorithm plaza overview. |
+| Service name | Yes | The service's title that will be displayed in the openEO algorithm plaza. |
+| Summary | No | Short description of the service. The summary will be visible in the openEO algorithm plaza overview.|
+| Description | No | The service description, begin with an algorithm overview, followed by a brief methodology. It is helpful to list required parameters and include examples and output images for user guidance.|
+| Avatar | No | URL to an image that can be used an avatar of the service. The avatar will be visible in the openEO algorithm plaza overview. |
 
 ##### Step 3: Additional Sections
 
-In addition to the basic information mentioned above, you can provide more details on your service by adding additional sections. The following sections are available:
+In addition to the basic information mentioned above, user can provide more details on their service by adding additional sections. The following sections are available:
 
-* Parameters: You can list all the parameters that should be fed to the service to execute it. You can specify the name, type, description, and default value for each parameter.
-* Usage example(Python code): You can provide a Python code example to demonstrate how to use your service. The code should give a complete example of how to use your service, including the use of the parameters and the namespace. 
-* Results: You can briefly describe your service's results and mention the output format of the result.
-* Cost Estimation: You can give a user an idea of the resource consumption and time required to run your service for a given input.
-* References: You can provide a list of references to publications, websites, or other resources relevant to your service. Provide details on resource consumption, processing time, and output format to help users effectively understand and utilize your service.
+* Parameters: A list of all the parameters that should be fed to the service to execute it. The name, type, description, and default value for each parameter should be specified.
+* Usage example(Python code): A Python code example to demonstrate how to use the service. The example should include the service's input parameters and the expected output.
+* Results: Service's results and it's output format should be mentioned.
```

Comment:
write is full out

Reply:
Sorry I didn't quite understand this comment. How do you mean write full out?